src/libraries/mysterium-client/client-log-subscriber.js

```
@@ -65,25 +81,18 @@ class ClientLogSubscriber {
       notifyOnErrorSubscribers(prependWithCurrentTime(prependWithSpace(data)))
     })
 
+    // not all OSs have a system file
+    if (!fs.existsSync(this._systemFilePath)) {
```

So if windows doesn't have this file, we should not pass `systemFilePath = '/var/log/system.log'` value - error seems to be in `mysteriumClient.config`.
As for cases when we are expecting system.log to be present, but it's not - we shouldn't ignore this error, we should report this to Sentry to be aware whether this happens often.

---

fixed